docs/deployments/api-configuration.md

```diff
@@ -99,6 +102,7 @@ See additional documentation for [autoscaling](autoscaling.md), [compute](comput
     model: <string>  # S3 path to an exported model (e.g. s3://my-bucket/exported_model.onnx) (required)
     config: <string: value>  # arbitrary dictionary passed to the constructor of the Predictor (optional)
     python_path: <string>  # path to the root of your Python folder that will be appended to PYTHONPATH (default: folder containing cortex.yaml)
+    image: <string> # optional image to override the configured cluster serving image (e.g. cortexlabs/onnx-serve[-gpu])
```

Comment:
nit: "# docker image to use for the Predictor (default: cortexlabs/onnx-serve[-gpu])"

Reply:
Done!